src/harness.js

```diff
@@ -60,6 +61,7 @@ class Harness {
     this.options = _.defaults(options, {
       verbose: false,
       logFormErrors: true,
+      excludeWarningsFromConsoleErrors: false,
```

Comment:
do you think these changes are still valuable? are people just trying to exclude that one warning or is there an actual broad need?